packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -169,6 +170,12 @@ class DropdownButton extends React.Component {
       theme.className,
     );
 
+    let buttonAriaLabel;
+    if (customProps && customProps['aria-label']) {
+      buttonAriaLabel = customProps['aria-label'];
+      delete customProps['aria-label'];
+    }
```

Comment:
Is there a reason we're doing this instead of creating an `ariaLabel` prop like in other components?

```suggestion
    let buttonAriaLabel;
    if (customProps && customProps['aria-label']) {
      buttonAriaLabel = customProps['aria-label'];
      delete customProps['aria-label'];
    }
```

Reply:
Making this change so that the aria-label added by the user applies to the button not the wrapping div